src/terminal.c

```diff
@@ -268,6 +278,49 @@ terminal_setflags(EditLine *el)
 libedit_private int
 terminal_init(EditLine *el)
 {
+	char tinfo_so[TC_BUFSIZE];
+	int major_version;
+
+	for (major_version = NCURSES_HIGH_VERSION; major_version >= NCURSES_LOW_VERSION; major_version--){
+		sprintf (tinfo_so, "libtinfo.so.%d", major_version);
+		if ((dl_handle = dlopen (tinfo_so, RTLD_LAZY)) != NULL)
+			break;
+	}
+
+	if (dl_handle == NULL){
+		fprintf (stderr, "ERROR: Cannot load ncurses library.\n");
```

Comment:
I think you need to tell the user the exact library name you need. So it would be nice to change the error message as the following.
"ERROR : Cannot load tinfo library. Please install tinfo library.\n"

Reply:
I thought that installing **ncurses** would also install tinfo.
I'll change that, thank you for your comment!